src/main/resources/assets/goodgenerator/data/essentia.json

```diff
@@ -326,57 +326,27 @@
     },
     {
       "name": "Radio",
-      "fuelValue": 238000,
+      "fuelValue": 456300,
       "category": "RADIATION",
       "consumeCeo": 1.0
     },
-    {
-      "name": "Custom1",
-      "fuelValue": 300000,
-      "category": "THERMAL",
-      "consumeCeo": 7.0
-    },
-    {
-      "name": "Custom2",
-      "fuelValue": 1,
-      "category": "NORMAL",
-      "consumeCeo": 0.0
-    },
-    {
-      "name": "Custom3",
-      "fuelValue": 217000,
-      "category": "NORMAL",
-      "consumeCeo": 0.0
-    },
-    {
-      "name": "Custom4",
-      "fuelValue": 118000,
-      "category": "RADIATION",
-      "consumeCeo": 0.5
-    },
-    {
-      "name": "Custom5",
-      "fuelValue": 120000,
-      "category": "VICTUS",
-      "consumeCeo": 0.6
-    },
     {
       "name": "Luxuria",
       "fuelValue": 79200,
       "category": "SPRITE",
-      "consumeCeo": 3.7
+      "consumeCeo": 70.0
```

Comment:
Christ, what's this about?

Reply:
It's a "fluid consumption rate" rather than a "burn rate increase multiplier", so essentially every increase after 1 is a nerf